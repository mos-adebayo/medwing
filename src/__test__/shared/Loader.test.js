import React from "react";
import ReactDOM from "react-dom";
import renderer from 'react-test-renderer';
import Loader from "../../components/shared/Loader";

test('Loader components snapshot', () => {
    const component = renderer.create(
        <Loader />,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});

test('Loader component renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Loader/>, div);
    ReactDOM.unmountComponentAtNode(div);
});
