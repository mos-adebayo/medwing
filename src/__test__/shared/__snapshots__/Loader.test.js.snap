// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader components snapshot 1`] = `
<div
  className="progress"
>
  <div
    className="indeterminate"
  />
</div>
`;
